live-build/misc/ansible-roles/appliance-build.minimal-common/tasks/main.yml

```diff
@@ -76,3 +76,8 @@
   delay: 30
   register: result
   until: result is succeeded
+
+#
+# CRA on Linux requires that the engine UUID is stored in a file
+#
+- shell: dmidecode -s system-uuid | awk '{print tolower($0)}' > /etc/engine-uuid
```

Comment:
Does `/etc/engine-uuid` need to be different for each engine? If we create the file here, I think it will be the same for every engine created from this build. Also, do we need the engine UUID to match system-uuid, or could we generate something really random using something like `cat /proc/sys/kernel/random/uuid` or `dd if=/dev/urandom bs=16 count=1 2>/dev/null | xxd -p | sed -E 's/(.{8})(.{4})(.{4})(.{4})(.{12})/\1-\2-\3-\4-\5/'`?

Reply:
Ah I see Seb and Pavel beat me to it.